### PR TITLE
some changes

### PR DESCRIPTION
**Pull request by hefee**
- make freebusy working
- make the server run as standalone
- do not randomly shutdown server on first user exit
- make datatests optional for build

### CODE REVIEW

File: kolabpopulated/build.py [hunk 1]
```diff
@@ -30,6 +30,9 @@ def main(dataset):
         # Wait for imap to become available on imaps://localhost:993
         time.sleep(5)
 
+        print "install php-Monolog"
+        docker("exec", container, "yum", "-y", "install", "php-Monolog", _out=process_output)
```
Comment (cmollekopf) [inline kolabpopulated/build.py]: php monolog is already installed in the base kolab image

File: testenv.py [hunk 2]
```diff
@@ -48,13 +46,22 @@ def build(options):
...
+        print("start " + dataset + " in background")
+    else:
+        print("start " + dataset + " " + clientconfigset)
+
+    cname = "{}:{}".format(settings.REPOSITORY, settings.populatedTag(dataset))
+    started = dockerutils.findContainer(cname) != ""
+    container = startContainer(cname, lambda: kolabpopulated.run.main(dataset, standalone))
+    if not standalone:
+        kontact.run.main(container, clientconfigset)
+        if not started:
```
Comment (cmollekopf) [inline testenv.py:62]: This should probably be "if started:" ?

File: kolabpopulated/build.py [hunk 3]
```diff
@@ -49,6 +52,9 @@ def main(dataset):
         docker("exec", container, "cp", "/data/roundcubemail/kolab_auth.inc.php", "/etc/roundcubemail/", _out=process_output)
         docker("exec", container, "cp", "/data/roundcubemail/password.inc.php", "/etc/roundcubemail/", _out=process_output)
 
+        print "restart httpd"
+        docker("exec", container, "service", "httpd", "restart", _out=process_output)
```
Comment (cmollekopf) [inline kolabpopulated/build.py]: I don't think this is necessary...

File: testenv.py (hunk 2 above)
Comment (hefee) [inline testenv.py:62]: The idea is:

if the server is not started we start it and kill it afterwards
the started bool indicates if the server was already started.

so "not started" is true if the server was not running before.

File: kolabpopulated/build.py (hunk 3 above)
Comment (hefee) [inline kolabpopulated/build.py]: when there is no need to install php-monolog than a restart is not need too.

**Comment by hefee** [comment]
updated pullrequest and remove parts that install php-monolog and restarts httpd

**Pull request merged**